build.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+import * as esbuild from "esbuild";
+import { glob } from "glob";
+
+const extensions = await glob("src/extensions/**.ts");
+
+console.log(`Compiling extensions: ${extensions}`);
+
+await esbuild.build({
+  entryPoints: ["src/index.ts"].concat(extensions),
+  bundle: true,
+  outdir: "build/",
+  format: "esm",
+  target: "esnext",
```

Comment:
Since we target the node minimum version `16.14.0` and up, I think we should set this `target` to `"node16.14.0"` (or alternatively set it to `"es2021"`).

Reply:
`"es2021"` doesn't work, which is why I went with `"esnext"`. `"node14.14.0"` works, and is probably the better choice; updated that.